agents/agents-tools/src/commonMain/kotlin/ai/koog/agents/core/tools/annotations/LLMMax.kt

```diff
@@ -0,0 +1,20 @@
+package ai.koog.agents.core.tools.annotations
+
+import kotlinx.serialization.SerialInfo
+
+
+/**
+ * Annotation used to specify the maximum allowable numeric value for properties, parameters,
+ * return values, or other entities provided to LLMs.
+ *
+ * This annotation can be applied to properties, classes, types, value parameters, or functions
+ * to enforce a constraint that the value must not exceed the specified maximum value.
+ *
+ * @property max The maximum allowable value.
+ */
+@SerialInfo
+@Target(
+    AnnotationTarget.PROPERTY,
+    AnnotationTarget.VALUE_PARAMETER,
+)
+public annotation class LLMMax(val max: Int)
```

Comment:
For this and other annotations:

1. Since they are applicable only for structured output (unlike `@LLMDescription`, which is also used for tools), let's not put these in tools module. There's already an `annotations` package in `prompt-structure` module, let's put it there. 
2. Also, the naming is a bit confusing, since it doesn't state it explicitly that it's for a JSON generation only. Can we maybe expand a bit, something like: `LLMStructureMax`